requirements.txt

```diff
@@ -1,3 +1,3 @@
-Django==3.2.5
 beautifulsoup4==4.9.1
+Django==3.2.13
```

Comment:
In the App Json, `3.2.12` is the one being installed. Should we change this version to match that?

Reply:
@shiyingtu Do we need to upgrade the library?

Reply:
Yes. Django 3.2.12 is still rejected by whitesource scan

Reply:
Resolve the thread by updating the app json and the wheel